packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -183,7 +183,10 @@ async function _getDashboardStats(app) {
   dashboard.nHosts = hosts.length
   dashboard.nHostsEol = nHostsEol
 
-  if (await app.hasFeatureAuthorization('LIST_MISSING_PATCHES')) {
+  const hasFeatureAuthorizationListMissingPatchesPromise = app.hasFeatureAuthorization('LIST_MISSING_PATCHES')
+  const remotesInfoPromise = app.getAllRemotesInfo()
```

Comment:
You cannot leave a dangling promise like that, uncaught rejections are forbidden.